sysfs/class_infiniband.go

```diff
@@ -175,9 +175,9 @@ func parseState(s string) (uint, string, error) {
 
 // Parse rate (example: "100 Gb/sec (4X EDR)") and return it as bytes/second
 func parseRate(s string) (uint64, error) {
-	parts := strings.Split(s, "Gb/sec")
+	parts := strings.SplitAfterN(s, " ", 2)
```

Comment:
should we call to lower to make sure it is gb/s ?

Reply:
I don't think we really care what the string is, based on the [kernel source](https://github.com/torvalds/linux/blob/master/drivers/infiniband/core/sysfs.c#L293-L295) it's always going to represent Gb/sec.

I'm not sure what's up with why my card driver, I can't find the upper-case version anywhere in the source.